stardog/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: stardog
-version: 0.5.0
+version: 0.5.1
```

Comment:
Since this is a new feature and not a bugfix I'd change that to 0.6.0

Reply:
Yes, good point. I have made the update.